### PR TITLE
clarify credential caching

### PR DESCRIPTION
**Pull request by glicht**
## Status
Ready

## Related Issues
related: https://github.com/demisto/etc/issues/27984

## Description
Clarify regarding credential caching

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by glicht** [comment]
@orius123 can you fill in how long credentials are cached and how often the cache is refreshed

**Comment by orius123** [comment]
Can be set with "vault.module.cache.expire", it is in minutes
Default is 10 minutes

File: content-repo/extra-docs/articles/managing-credentials.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ Cortex XSOAR Credentials enables you to centrally manage credentials, which incl
 
 ## Configure an External Credentials Vault
 
-Cortex XSOAR integrates with external credential vaults, which enables you to use them in Cortex XSOAR without hard coding or exposing them. The credentials are not stored in Cortex XSOAR, rather, the integration fetches the credentials from the external vault when called.
+Cortex XSOAR integrates with external credential vaults, which enables you to use them in Cortex XSOAR without hard coding or exposing them. The credentials are not stored in Cortex XSOAR, rather, the integration fetches the credentials from the external vault when called. The credentials are fetched and cached in-memory. They are passed to integrations that are executed as part of the integrations params.
```
Comment (orius123) [inline content-repo/extra-docs/articles/managing-credentials.md:44]:
```suggestion
Cortex XSOAR integrates with external credential vaults, which enables you to use them in Cortex XSOAR without hard coding or exposing them. The credentials are not stored in Cortex XSOAR, rather, the integration fetches the credentials from the external vault when called. The credentials are fetched and cached in-memory for 10 minutes by default. They are passed to integrations that are executed as part of the integrations params.
```

**Comment by glicht** [comment]
@orius123 please add data about how to change the fetch timeout

File: content-repo/extra-docs/articles/managing-credentials.md [hunk 2]
```diff
@@ -41,7 +41,9 @@ Cortex XSOAR Credentials enables you to centrally manage credentials, which incl
 
 ## Configure an External Credentials Vault
 
-Cortex XSOAR integrates with external credential vaults, which enables you to use them in Cortex XSOAR without hard coding or exposing them. The credentials are not stored in Cortex XSOAR, rather, the integration fetches the credentials from the external vault when called.
+Cortex XSOAR integrates with external credential vaults, which enables you to use them in Cortex XSOAR without hard coding or exposing them. The credentials are not stored in Cortex XSOAR, rather, the integration fetches the credentials from the external vault when called. The credentials are fetched and cached in-memory for 10 minutes by default. They are passed to integrations that are executed as part of the integration params.
```
Comment (orius123) [inline content-repo/extra-docs/articles/managing-credentials.md:44]:
```suggestion
Cortex XSOAR integrates with external credential vaults, which enables you to use them in Cortex XSOAR without hard coding or exposing them. The credentials are not stored in Cortex XSOAR, rather, the integration fetches the credentials from the external vault when called. The credentials are fetched and cached in-memory for 10 minutes by default, can be modified with the `vault.module.cache.expire` configuration key (set to 0 will disable caching). They are passed to integrations that are executed as part of the integration params.
```

**Comment by kirbles19** [comment]
@glicht Done.

**Pull request merged**